tutorials/ksample.py

```diff
@@ -164,6 +164,77 @@
 # Any addition, if the bias variant of the test statistic is required, then the ``bias``
 # parameter can be set to ``True``. In general, we do not recommend doing this.
 # Otherwise, these tests runs like :ref:`any other test<general indep>`.
+########################################################################################
+# Smooth Characteristic Function Test
+#
+# The Smooth Characteristic Function Test, or Smooth CF, is a form of non-parametric two-sample
+# tests. The Smooth CF test utilizes smoothed empirical characteristic functions to represent
+# two data distributions. Characteristic functions completely define the probability distribution
+# of a random variable. In hypothesis testing, it is useful to estimate characteristic functions
+# for given data. However, empirical characteristic functions can be very complex and therefore
+# expensive to compute. The smooth characteristic function can serve as a heuristic in place of
+# the empirical function which is much faster w.r.t. computation times.
+#
+# .. note::
+#
+#   The Smooth Characteristic Function Test can only be used in two-sample testing cases.
+#
+# .. note::
+#
+#   :Pros: - Very fast computation time
+#          - Faster than current, state-of-the-art quadratic-time kernel-based tests
+#   :Cons: - Only supports two-sample testing
+#
+import numpy as np
+from hyppo.ksample import SmoothCFTest
+
+
+num_samples = 500
+dimensions = 10
+X = np.random.randn(num_samples, dimensions)
+Y = np.random.randn(num_samples, dimensions)
+print(X.shape, Y.shape)
+
+smoothCF = SmoothCFTest()
+stat, p_val = smoothCF.test(X, Y)
+#
+#
+#
+########################################################################################
+# Mean Embedding Test
+# --------------------------------------------
+# The mean embedding test is another non-parametric two-sample statistical test. This test
```

Comment:
link back to your reference documentation

Reply:
@sampan501 do youmean the paper or my code?

Reply:
Well there should a place in the reference documentation where your test shows up, which it currently doesn't. So I would fix that first, and then link to that page

Reply:
i.e. [this](https://hyppo.neurodata.io/tutorials/independence.html#pearson-s-correlation-multivariate-variants) links to here automatically with this hyperlink: [hyppo.independence.CCA](https://hyppo.neurodata.io/api/generated/hyppo.independence.CCA.html#hyppo.independence.CCA)

Reply:
links added